pages/result.tsx

```diff
@@ -0,0 +1,19 @@
+import { Grid, Text } from '@chakra-ui/react';
+
+import ResulData from '@packages/components/ResultData';
+
+import type { NextPage } from 'next';
+
+const ResultPage: NextPage = () => {
+  return (
+    <Grid h="100vh" w="100vw" justifyContent="center">
+      <Text fontSize="30px" textAlign="center" marginTop="100px">
+        {' '}
+        Resultado da Votação
+      </Text>
+      <ResulData />
```

Comment:
i think that @frattezi ment [here](https://github.com/Joel-leal/voting-system/pull/49/files#r912979572) to put [this](https://github.com/Joel-leal/voting-system/pull/49/files#diff-2b5738d86f7a59175756c4a91dc2107b42c03cd517b2b5950e15e7d8dbb28024R14-R18) here